ui/packages/app/src/project_setting/form/ProjectForm.js

```diff
@@ -192,26 +121,21 @@ const ProjectForm = () => {
             <EuiDescribedFormGroup
               title={<h3>Stream</h3>}
               description="Product stream the project belongs to">
-              <EuiFormRow isInvalid={!isValidStream} error={streamError}>
-                <EuiComboBoxSelect
-                  value={project.stream}
-                  options={streamOptions}
-                  onChange={onStreamChange}
-                  onCreateOption={onStreamChange}
-                />
-              </EuiFormRow>
+              <Stream
+                stream={project.stream}
+                setStream={setStream}
+                setIsValidStream={setIsValidStream}
+              />
```

Comment:
How is this `Stream` component supposed to know if the value of `isValidStream` to pass to the nested `EuiFormRow` if you don't pass it as a prop? 😮